bindings/matrix-sdk-ffi/src/identity_status_change.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright 2024 The Matrix.org Foundation C.I.C.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#[derive(uniffi::Record)]
+pub struct IdentityStatusChange {
+    /// The user ID of the user whose identity status changed
+    pub user_id: String,
+
+    /// The new state of the identity of the user. One of:
+    ///
+    /// * Verified
+    /// * Pinned
+    /// * PinViolation
+    /// * PreviouslyVerified
+    ///
+    /// See
+    /// [`matrix_sdk_crypto::identities::room_identity_state::RoomIdentityState`]
+    /// for details.
+    pub changed_to: String,
```

Comment:
Why using a `String` here and not `IdentityState`?

Reply:
Hmm. I thought I needed to use basic types like `String` here. Looking at whether it's OK to use `IdentityState`.

Reply:
> I had to wrap it because `IdentityState` is in the crypto layer so doesn't know about `uniffi`. Fixed in [c2716b9](https://github.com/matrix-org/matrix-rust-sdk/commit/c2716b9366e4358332ba1ab249266d417b988e1a)

That's not quite true, the crypto crate depends on `uniffi` and has a `uniffi` feature, this way we can expose simpler types without any additional wrappers:

https://github.com/matrix-org/matrix-rust-sdk/blob/59d3608c32c94b7bf210a9bee1ba2733d37b2cec/crates/matrix-sdk-crypto/src/session_manager/group_sessions/share_strategy.rs#L36-L40

Reply:
Ah great, thanks - I'll do that!

Reply:
Done in 1a2b0f839d and 7128c54f61